tests/test-cases/Group1-Docker-Commands/1-32-Docker-Plugin.robot

```diff
@@ -41,9 +41,7 @@ Docker plugin disable
     Should Contain  ${output}  does not yet support plugins
 
 Docker plugin inspect
-    ${rc}  ${output}=  Run And Return Rc And Output  docker1.13 %{VCH-PARAMS} plugin inspect test-plugin
-    Should Be Equal As Integers  ${rc}  1
-    Should Contain  ${output}  does not yet support plugins
+    Pass execution  Test not implemented till plugin support is added
```

Comment:
@mhagen-vmware There's no GitHub issue for `docker plugin` work because it's out of scope for upcoming releases. In the interim, I feel that saying `Test not implemented till plugin support is added` is sufficient to grab attention.